packages/utils/style/colors.ts

```diff
@@ -43,7 +42,10 @@ export const neutralA3 = "#A3A3A3";
 export const transparentColor = "transparent";
 
 export const additionalRed = "#FFAEAE";
+export const additionalRed10 = "rgba(244, 111, 118, 0.1)";
```

Comment:
I know that in the codebase, we use `withAlpha` function directly in the code, so can you fit with that ?